DigitalLearningSolutions.Data/Services/FrameworkService.cs

```diff
@@ -841,13 +841,9 @@ FROM    FrameworkCollaborators
 
         public void RemoveCollaboratorFromFramework(int frameworkId, int id)
         {
-            var adminId = (int?)connection.ExecuteScalar(
-                @"SELECT AdminID FROM FrameworkCollaborators WHERE  (FrameworkID = @frameworkId) AND (ID = @id)",
-                new { frameworkId, id }
-            );
             connection.Execute(
-                @"DELETE FROM  FrameworkCollaborators WHERE (FrameworkID = @frameworkId) AND (ID = @id);UPDATE AdminUsers SET IsFrameworkContributor = 0 WHERE AdminID = @adminId AND AdminID NOT IN (SELECT DISTINCT AdminID FROM FrameworkCollaborators);",
-                new { frameworkId, id, adminId }
+                @"DELETE FROM FrameworkReviews WHERE FrameworkCollaboratorID=@id and FrameworkID=@frameworkId;DELETE FROM FrameworkCollaborators WHERE (FrameworkID = @frameworkId) AND (ID = @id);",

```

Comment:
If framework reviews exist. We don't want to delete them. I think we need a soft delete in this instance.

I suggest we close this PR without merging and create a new ticket to cover the work required for soft-delete. Does that sound okay?

Reply:
Yes that sounds good.